### ISSUE TITLE
Verified website, github, twitter in header

### ISSUE THREAD

**Issue by patak-dev** [elk-zone/elk#155]
@antfu thinking about how to implement this with the current UI. Maybe a primary color check icon next to the link? Or just use a primary color for the link when it is verified? If we don't want to use green, I think we may need the tick

https://mas.to/@addyosmani@indieweb.social

https://elk.zone/@addyosmani@indieweb.social

**Comment by boehs** [comment]
I don't see how to fit the tick into here

**Comment by ZMYaro** [comment]
I think including the tick, or some other indicator that is not just color, is necessary as an accessibility issue.  Additionally, as I understand it, there are people who use Elk in grayscale mode, to whom a color indicator would not be clear?

**Comment by wakest** [comment]
@ZMYaro thats a good point about the accessibility issue... In this case Green is an awful color to use then...

**Comment by ZMYaro** [comment]
@wakest Green is fine to use as long as it is combined with a non-color indicator (like the checkmark).

**Comment by patak-dev** [comment]
One possible idea is to make the meta info bigger. It is hard to add more than just color with the current design, but it would be easier if we have a bit more air. I don't know if we should go to one line per meta, but maybe that could be revisited? I think the meta info is quite important, and we are having it small and in secondary color currently.

<img width="640" alt="image" src="https://user-images.githubusercontent.com/583075/215887826-8a53d60e-d0f3-4ebe-ae95-d2fe4f2342ae.png">

**Comment by ZMYaro** [comment]
On possibility is something like what I mocked up for #1101, where we could have the label small and attached to the link, and then it would be less of an issue if the icon became a checkmark when verified.
![image](https://user-images.githubusercontent.com/763405/212782366-7c5e4a9f-0108-4017-bf3a-042eabfe9f35.png)
![image](https://user-images.githubusercontent.com/763405/212782305-a6ecb8ff-dffe-4227-b989-74b5bb8bc184.png)

**Comment by ZMYaro** [comment]
Respectfully, I do not think this issue should be closed yet.  I think a more clear indicator should be used than an outline alone.  I can try to take a stab at an approach sometime this weekend if I have time.

**Comment by boehs** [comment]
Go for it

**Comment by gBasil** [comment]
*Maybe* something like this?
<img width="256" alt="image" src="https://user-images.githubusercontent.com/57513430/217299308-c06dca81-34bd-473f-9bd2-238a7788ad09.png">
We could also just format metadata like the default Mastodon web client does.

**Comment by boehs** [comment]
Seems like a good improvement! Please open a PR for this ❤️

**Comment by gBasil** [comment]
> Seems like a good improvement! Please open a PR for this ❤️

Would it be better for this be the tint color or green no matter the theme?

**Comment by boehs** [comment]
@patak-dev might have stronger opinions, but I'm inclined to theme colour

**Comment by gBasil** [comment]
Few implementation questions:
1. In the screenshot, I added some padding. I don't know if that's alright with you all.
2. How is this going to be applied to the verified links that aren't placed at the bottom? Example profile: https://elk.zone/tech.lgbt/@nachoqt I'm still going to push for recreating Mastodon's style, I really think that's the best way to go. It's potentially less confusing, and the current implementation doesn't really fit.
<img width="162" alt="image" src="https://user-images.githubusercontent.com/57513430/217419592-054d94b7-c5c0-4d90-84dc-d28b38f601f5.png">


Anyway, it's just these styles, which I'm posting here for backup before the questions are addressed:
```css
/* On the container div */
border-color: var(--c-primary);
background: var(--c-dark-primary-fade);
/* Magic numbers that make it look right */
padding: 1px 4px;
padding-right: 6px;

/* On the icon */
color: var(--c-primary);
```
<img width="104" alt="image" src="https://user-images.githubusercontent.com/57513430/217418616-384d09cf-0e58-4cd2-b60f-756728a0a587.png">

**Comment by boehs** [comment]
> In the screenshot, I added some padding. I don't know if that's alright with you all.

Yeah, that's great. Make sure that padding is on the other links as well (I forgot to do that in my PR)

> How is this going to be applied to the verified links that aren't placed at the bottom?

Honestly, unsure :/. Maybe get feedback in the UI channel 🙏🏻

**Comment by gBasil** [comment]
> Honestly, unsure :/. Maybe get feedback in the UI channel 🙏🏻

In the Discord guild?

**Comment by boehs** [comment]
> In the Discord guild?

Yeah ❤️

**Comment by ZMYaro** [comment]
Since they are a little bit buried in the PR UI, here are some screenshots of what my current proposal looks like:
This is how it looks on a few profiles:
![image](https://user-images.githubusercontent.com/763405/218036383-20f8bf16-ee53-43b3-ab49-884d2e00036d.png)
![image](https://user-images.githubusercontent.com/763405/218036629-073fbb03-9d8f-484d-8957-db0517f2452a.png)
![image](https://user-images.githubusercontent.com/763405/218036746-3811f475-a852-47e6-9f13-355b53eb7227.png)
![image](https://user-images.githubusercontent.com/763405/218037112-c1068237-4ead-4e3f-aee5-9823674946cf.png)

**Comment by gBasil** [comment]
Looks great! Just my two cents, I feel like verified links should be emphasized more. Also, how are non-standard metadata titles handled?

**Comment by ZMYaro** [comment]
> I feel like verified links should be emphasized more.
My own opinion is that the checkmark + outline is sufficient, but we could try making them the accent color if the consensus is that would be better.

>Also, how are non-standard metadata titles handled?
Links with non-standard names get a chip with a standard link icon (like the link to my Linktree in the first screenshot), and other fields with non-standard names display as they do right now.

**Comment by ZMYaro** [comment]
Here is what a verified link looks like if we make it entirely the accent color:
![image](https://user-images.githubusercontent.com/763405/218821951-6203f84f-8108-4eb7-82fa-a3335ae60e6b.png)
![image](https://user-images.githubusercontent.com/763405/218822278-f37f8497-830d-4d23-82ac-8d9758f2bf0d.png)

**Comment by ZMYaro** [comment]
And then this is the same implementation, but with the circled checkmark icon:
![image](https://user-images.githubusercontent.com/763405/222635154-5ecf19ac-91c7-4f59-ac90-1fad2d930f9a.png)
![image](https://user-images.githubusercontent.com/763405/222635230-1a3c170a-bf60-4fcf-bcc3-152edb72ad8c.png)

**Comment by boehs** [comment]
Strong support for the above from me!

**Comment by wakest** [comment]
[ZMYaro](https://github.com/ZMYaro)'s approach looks great
